reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -1013,7 +1013,7 @@
 	 * to be cancelled and the resulting Mono to immediately error or complete, respectively.
 	 *
 	 * <p>
-	 * <img class="marble" src="https://raw.githubusercontent.com/reactor/reactor-core/v3.1.3.RELEASE/src/docs/marble/whent.png" alt="">
+	 * <img class="marble" src="https://raw.githubusercontent.com/reactor/reactor-core/v3.1.3.RELEASE/src/docs/marble/zip1.png" alt="">
```

Comment:
missed this `v3.1.3.RELEASE` to `master`

Reply:
Actually this one already exists in `v3.1.3.RELEASE`

Reply:
:+1: